my-mill-postgui.hal

```diff
@@ -8,14 +8,14 @@
 
 
 # The next two lines are only needed if the pins had been connected before
-unlinkp iocontrol.0.tool-change
-unlinkp iocontrol.0.tool-changed
+#unlinkp iocontrol.0.tool-change
+#unlinkp iocontrol.0.tool-changed
```

Comment:
If users copy this file as is / as per the readme, we end up unlinking the connections made in `probe_screen.hal`. I haven't checked, but I suspect the ordering of when different things are loaded has changed in 2.8.